cg/meta/workflow/balsamic.py

```diff
@@ -316,27 +296,55 @@ def get_verified_gender(sample_data: dict) -> Union[Gender.FEMALE, Gender.MALE]:
             LOG.error(f"Unable to retrieve a valid gender from samples: {sample_data.keys()}")
             raise BalsamicStartError
 
+    def get_verified_samples(self, case_id: str, sample_data: dict) -> Dict[str, str]:
+        """Return a verified tumor and normal sample dictionary."""
+        tumor_name: str = self.status_db.get_sample_name_by_type(
+            case_id=case_id, sample_type=SampleType.TUMOR
+        )
+        tumor_path: str = self.get_verified_sample_path(
+            sample_data=sample_data, sample_type=SampleType.TUMOR
+        )
+        normal_name: str = self.status_db.get_sample_name_by_type(
+            case_id=case_id, sample_type=SampleType.NORMAL
+        )
+        normal_path: str = self.get_verified_sample_path(
+            sample_data=sample_data, sample_type=SampleType.NORMAL
+        )
+
+        if not tumor_path and not normal_path:
+            LOG.error(f"No tumor or normal samples were found for case {case_id}")
+            raise BalsamicStartError
+
+        if normal_path and not tumor_path:
+            LOG.warning(
+                f"Only a normal sample was found for case {case_id}. "
```

Comment:
This is good, but in production we do not check all warnings unless there's something weird going on. Maybe it would be nice to have this information in the delivery report. We could specify there that a normal sample has been treated as a tumor and that this is NOT validated in balsamic and at the user own risk. Or something like that. But this could be added later in a different PR.

Reply:
I would say that this PR is a hot fix and should be reverted as soon as we figure out why the customers need to run Balsamic analyses on normal samples.

Now the question is, should it even be picked by the automation or should we only allow these cases to be started manually? What do you think?

Reply:
I don't see why we would revert this, this might come back again in the future. But I agree that we would need to take care of the automation. I would say we do not want this to be picked by the automation UNLESS there are many of these types of cases (which would result in a lot of manual work for people in prod). But possibly @karlnyr knows better

Reply:
I've added a `--force-normal` flag, which will prevent these analyses from starting automatically. We can also easily remove it if we start getting too many of these cases.